app/services/recover/validate_request_token.rb

```diff
@@ -1,28 +0,0 @@
-module Recover
-  class ValidateRequestToken
-    include ActiveModel::Model
-    include Idv::RecoverTokenValidator
```

Comment:
This concern can be removed:

https://github.com/18F/identity-idp/blob/stevegsa-rm-ial2-recovery/app/validators/idv/recover_token_validator.rb